File: app/src/main/java/com/alvarlagerlof/koda/Projects/ProjectsObjectData.java
package com.alvarlagerlof.koda.Projects;

/**
 * Created by alvar on 2016-07-02.
 */
class ProjectsObjectData {
    String privateID;
    String publicID;
    String title;
    String updated;
    String description;
    Boolean isPublic;
    String code;


    ProjectsObjectData(String privateID,
                       String publicID,
                       String title,
                       String updated,
                       String description,
                       Boolean isPublic,
                       String code) {

        this.privateID   = privateID;
        this.publicID    = publicID;
        this.title       = title;
        this.updated     = updated;
        this.description = description;
        this.isPublic    = isPublic;
        this.code        = code;

    }

}

File: app/src/main/java/com/alvarlagerlof/koda/Projects/ProjectsObjectData.kt
package com.alvarlagerlof.koda.Projects

/**
 * Created by alvar on 20 16-07-02.
 */
data class ProjectsObjectData(val privateID: String,
                              val publicID: String,
                              val title: String,
                              val updated: String,
                              val description: String,
                              val isPublic: Boolean?,
                              val code: String)